packages/react-scripts/template/README.md

```diff
@@ -936,6 +937,50 @@ Install the Surge CLI if you haven't already by running `npm install -g surge`.
 
 Note that in order to support routers that use html5 `pushState` API, you may want to rename the `index.html` in your build folder to `200.html` before deploying to Surge. This [ensures that every URL falls back to that file](https://surge.sh/help/adding-a-200-page-for-client-side-routing).
 
+## Fork instead of eject
+
+If you want to customize the default configuration slightly (e.g. add CSS Modules, SASS, decorators…), while still having all future updates and a one dependency, you can fork this repo and create your own configuration of `react-scripts` and use it with `create-react-app`.
+
+### Instruction
+
+1: Fork [create-react-app repo](https://github.com/facebookincubator/create-react-app).
+
+2: Change the name of `react-scripts` package to your new one, and also reset its version.
+```js
+// /packages/react-scripts/package.json
+{
+  "name": "my-react-scripts-fork",
+  "version": "0.0.1",
+  …
+}
+```
+
+3: Make your changes inside `react-scripts` package. For example, add CSS Modules:
+```js
+// /packages/react-scripts/config/webpack.config.dev.js
+…
+{
+  test: /\.css$/,
+  loader: 'style!css?modules=1!postcss'
+},
+…
+```
+
+4: [Publish](https://docs.npmjs.com/getting-started/publishing-npm-packages) your customized `react-scripts` package to npm. Remember to cd into `/packages/react-scripts/` before you publish.
+```sh
+$ cd packages/react-scripts
+$ npm publish
```

Comment:
I tried to follow your instructions, but this will not publish a working `react-scripts` package. The package needs to be published with `create-react-app`'s npm script `npm run publish`, as this will do some cleanup tasks before publishing `react-scripts` to npm. Check this out for more info: https://github.com/facebookincubator/create-react-app/blob/master/CONTRIBUTING.md#cutting-a-release